src/navability/graphql/DataBlobs.jl

```diff
@@ -70,6 +70,50 @@ mutation sdk_adddataentry(\$userId: ResourceId!, \$robotId: ResourceId!, \$sessi
 }
 """
 
+GQL_ADDBLOBENTRY = """
+mutation sdk_addblobentry(
+  \$userId: String!
+  \$robotId: String!
+  \$sessionId: String!
+  \$variableLabel: String!
+  \$blobId: UUID!
+  \$dataLabel: String!
+  \$mimeType: String
```

Comment:
```suggestion
  \$size: Int!
  \$mimeType: String
```

Reply:
![Screenshot from 2023-02-10 11-39-35](https://user-images.githubusercontent.com/6412556/218183081-06142056-8e83-4e5c-8677-e3b07b903d22.png)

Reply:
I already have this local, but it wasn't working yet.